keylime/revocation_notifier.py

```diff
@@ -80,6 +80,12 @@ def stop_broker():
 
 
 def notify(tosend):
+    # python-requests internally uses either simplejson (preferred) or
+    # the built-in json module, and when it is using the built-in one,
+    # it may encounter difficulties handling bytes instead of strings.
+    # To avoid such issues, let's convert `tosend' to str beforehand.
+    tosend = json.__bytes_to_str(tosend)
```

Comment:
`__bytes_to_str` is marked as a private function with the `__` prefix. Can you change the name of the function to `bytes_to_str` if we are now using it outside the `json.py` file? 

Otherwise LGTM.

Reply:
Good point, I have updated it.